src/runtime/RuntimeError.zig

```diff
@@ -11,10 +11,7 @@ const std = @import("std");
 const Allocator = std.mem.Allocator;
 
 const context = @import("context.zig");
-const Activation = @import("./Activation.zig");
 const SourceRange = @import("./SourceRange.zig");
-const VirtualMachine = @import("./VirtualMachine.zig");
-
```

Comment:
Could you please make it so that the single empty line between imports and the fields are preserved?

Reply:
Right, I can do that.